pymc3/sampling.py

```diff
@@ -1283,8 +1321,9 @@ def sample_prior_predictive(samples=500, model=None, vars=None, random_seed=None
     values = draw_values([model[name] for name in names], size=samples)
 
     data = {k: v for k, v in zip(names, values)}
+    assert data is not None
 
-    prior = {}
+    prior: Dict[str, np.ndarray] = {}
```

Comment:
does this one break on python 3.5? I thought we could only do signature type checks...

Reply:
Oh, probably. Sorry. I'll fix that.

Reply:
Done.  See 99c7362d81bc729e0fa62de7039bb19b2a8d991d